### ISSUE TITLE
Info on authentication

### ISSUE THREAD

**Issue by nicolabeghin** [italia/api-padigitale2026-misura1.3.1-uni-afam#157]
Hi would it be possible to extend the README with the Authentication info? It's pretty hard to follow all the PDFs with cross references to other PDFs with and without technical details.. 😅

Also if any reference implementation is available including authentication (apart the [global one based on Go that I'm not sure it's 100% applicable to this specific domain](https://github.com/italia/developers-Italia-api))

Thanks!
nicola

**Comment by nicolabeghin** [comment]
Up

**Comment by denismarini** [comment]
Hi dear Nicola,

I'm sorry if I'm answering late, I hope we can help you now. The Authentication details are on PDND platform documentation https://docs.pagopa.it/interoperabilita-1/come-integrare-i-propri-servizi-su-pdnd-interoperabilita. 

I recommend you to evaluate this Open Source API Gateway shared in the web portal Deveolopers.it that 
 already implements the authentication protocol as a Framework https://developers.italia.it/it/software/link-it-govway-7218c1.html 
https://govway.org/

**Comment by nicolabeghin** [comment]
Would it be possible to share documentation on the suggested way to configure govway for PDND? thanks!

**Comment by denismarini** [comment]
Hi @nicolabeghin here the doc https://govway.org/documentazione/

Thanks!

**Comment by nicolabeghin** [comment]
Given the auth topic is non-trivial and everybody will need to address it at some point, would it be possible to provide something more apart from the homepage of the gateway? Thanks a lot
